### PR TITLE
MCH: implement custom inputs spec for raw decoder

### PR DESCRIPTION
**Pull request by aferrero2707**
The input dataspec might vary from the default "TF:MCH/RAWDATA", for example when a data sampling device is inserted between the raw proxy and the decoder.
    
This PR allows to specify the input dataspec as a command-line option, giving more flexibility to the DPL chain.
The documentation is updated accordingly.

The PR also introduces a simple DPL device to debug TimeFrames. The device prints the first and last RDH of each received TF. Useful for debugging data flow and data sampling.
    
There is also some improvement in the textual output of the digits sink.

### CODE REVIEW

File: Detectors/MUON/MCH/Workflow/src/raw-parser.cxx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
```
Comment (aphecetche) [inline Detectors/MUON/MCH/Workflow/src/raw-parser.cxx]: I guess this is a cut-and-paste from some other device ... ;-)

File: Detectors/MUON/MCH/Workflow/CMakeLists.txt [hunk 2]
```diff
@@ -34,6 +34,12 @@ o2_add_executable(
         COMPONENT_NAME mch
         PUBLIC_LINK_LIBRARIES O2::MCHWorkflow)
 
+o2_add_executable(
```
Comment (aphecetche) [inline Detectors/MUON/MCH/Workflow/CMakeLists.txt:37]: if that device is only intended for debug, I would make it crystal clear by adding -debug- (or test or whatever) to the executable name

**Review by aphecetche — CHANGES_REQUESTED** [review]

File: Detectors/MUON/MCH/Workflow/src/raw-parser.cxx (hunk 1 above)
Comment (aferrero2707) [reply Detectors/MUON/MCH/Workflow/src/raw-parser.cxx]: Fixed.

File: Detectors/MUON/MCH/Workflow/CMakeLists.txt (hunk 2 above)
Comment (aferrero2707) [reply Detectors/MUON/MCH/Workflow/CMakeLists.txt:37]: I renamed the code and executable as `raw-debug-workflow`. Is that ok for you?

**Review by aphecetche — APPROVED** [review]

**Pull request merged**